src/script/engines/lua.c

```diff
@@ -177,6 +325,20 @@ struct mScriptEngineContext* _luaCreate(struct mScriptEngine2* engine, struct mS
 	lua_getglobal(luaContext->lua, "require");
 	luaContext->require = luaL_ref(luaContext->lua, LUA_REGISTRYINDEX);
 
+	int status = luaL_dostring(luaContext->lua, _socketLuaSource);
+	if (status) {
+		mLOG(SCRIPT, ERROR, "Error in dostring while initializing sockets: %s\n", lua_tostring(luaContext->lua, -1));
+		lua_pop(luaContext->lua, 1);
+	} else {
+		lua_getglobal(luaContext->lua, "socket");
+		lua_getfield(luaContext->lua, -1, "ERRORS");
+		for (int i = 0; i < _mScriptSocketNumErrors; i++) {
```

Comment:
No definitions in `for` loops, though it's technically allowed in C99 I don't do it in mGBA code. I may consider changing this at some point though, especially now that I'm starting to move mGBA to C11.